corehq/apps/users/static/users/js/filtered_download.js

```diff
@@ -1,17 +1,19 @@
 /* global interpolate */
 hqDefine('users/js/filtered_download', function () {
-    $(function () {
-        var prevFilters = {
+    function getFilters() {
+        return {
             'role_id': $("#id_role_id").val(),
             'search_string': $("#id_search_string").val(),
+            'location_id': $("[name=location_id]").val(),
```

Comment:
why not actually give it a id and then search for it?

Reply:
no particular reason